### DISCUSSION TITLE
stderr: 'fatal: Unable to create '.git/index.lock': File exists.

### DISCUSSION THREAD

**Discussion by oma0256** [gitpython-developers/GitPython#1154] [Q&A] [answered]
I have a git script that fetches a file from a remote repository. Everything was working fine but then all of a sudden I'm running into this error below. Is their a way I can solve this using `GitPython`, any help will be much appreciated thanks.
```
GitCommandError
'git checkout origin/<branch_name> <filename>' returned exit code 128.
stderr: 'fatal: Unable to create '.git/index.lock': File exists.

if no other git process is currently running, this probably means a git process crashed in this repository earlier. Make sure no other git process is running and remove the file manually to continue.
```

**Comment by Byron** [comment, marked as answer]
It looks like the error is related to the git command-line tool, which might indeed have crashed, leaving a lock file preventing further access. This can possibly happen if the script is interrupted with Ctrl+C.

As there is nothing unusual about this happening, nor is there an indication that this is a bug in GitPython, I am closing this issue.
Please feel free to comment in the closed issue with follow-up questions or other comments that might make a bug reproducible.
